### PR TITLE
fix: default Prisma client generation location

### PR DESCRIPTION
**Pull request by janpio**
We changed this some time ago.

Might be worth looking through the rest of the documentation to find additional instances

### CODE REVIEW

**Comment by molebox** [comment]
Hey @janpio , i made the changes to all locations, have created conficts though due to my main being further ahead. Id be happy to solve these, that would mean either commiting to your branch or creating a new PR fromt he conflict resolution.

File: content/300-guides/300-upgrade-guides/01-upgrade-from-prisma-1/07-upgrading-a-rest-api.mdx [hunk 1]
```diff
@@ -126,7 +126,7 @@ Consider each occurrence of the Prisma client instance `prisma` and replacing wi
 
 ### 1.1. Adjusting the import
 
-Since Prisma Client 2 is generated into `node_modules/@prisma/client`, it is imported as follows:
+Since Prisma Client 2 is generated into `node_modules/.prisma/client`, it is imported as follows:
```
Comment (janpio) [inline content/300-guides/300-upgrade-guides/01-upgrade-from-prisma-1/07-upgrading-a-rest-api.mdx]: This sentence makes no sense this way any more - this now requires additional explanation. The code below only makes sense if the code comes from `@prisma/client` - but we generate into another folder and have additional logic in `@prisma/client` that it works anyway.

**Comment by janpio** [comment]
All these unrelated formatting changes should not be in this PR. I get that there is no way to undo this any more now realistically, but I strongly prefer to keep PRs limited to the actual relevant changes - especially if someone commits into one of my branches.

**Comment by janpio** [comment]
Looking at the conflicts, they are pretty trivial so you should be able to fix them via GH UI directly @molebox. I am missing the knowledge which of these links and link titles are not the correct ones.

**Comment by molebox** [comment]
> All these unrelated formatting changes should not be in this PR. I get that there is no way to undo this any more now realistically, but I strongly prefer to keep PRs limited to the actual relevant changes - especially if someone commits into one of my branches.

Are these prettier formatting changes you mean? Wasnt my intention to do any such changes, only the client paths.

My bad. In the future I'll open a new PR.

File: content/100-getting-started/02-setup-prisma/01-add-to-existing-project.mdx [hunk 2]
```diff
@@ -480,7 +480,7 @@ Notice that the [`@prisma/client` node module](../../concepts/components/prisma-
 npx prisma generate
 ```
 
-This command reads your Prisma schema and generates your Prisma Client library into `node_modules/@prisma/client`.
+This command reads your Prisma schema and generates your Prisma Client library into `node_modules/.prisma/client`.
```
Comment (janpio) [inline content/100-getting-started/02-setup-prisma/01-add-to-existing-project.mdx]:
```suggestion
This command reads your Prisma schema and generates your Prisma Client library:
```

The folder is mentioned both in the section before and after this one, so not necessary here.

File: content/200-concepts/050-overview/100-what-is-prisma/index.mdx [hunk 3]
```diff
@@ -95,9 +95,9 @@ The first step when using Prisma Client is installing the `@prisma/client` npm p
 npm install @prisma/client
 ```
 
-Installing the `@prisma/client` package invokes the `prisma generate` command, which reads your Prisma schema and _generates_ the Prisma Client code. The code will be located in `node_modules/@prisma/client`.
+Installing the `@prisma/client` package invokes the `prisma generate` command, which reads your Prisma schema and _generates_ the Prisma Client code. The code will be located in `node_modules/.prisma/client`.
```
Comment (janpio) [inline content/200-concepts/050-overview/100-what-is-prisma/index.mdx]: "The code will be located ..." should probably link to where we explain how this works in detail.

File: content/200-concepts/050-overview/100-what-is-prisma/index.mdx [hunk 4]
```diff
@@ -95,9 +95,9 @@ The first step when using Prisma Client is installing the `@prisma/client` npm p
...
 
-After you change your data model, you'll need to manually re-generate Prisma Client to ensure the code inside `node_modules/@prisma/client` get updated:
+After you change your data model, you'll need to manually re-generate Prisma Client to ensure the code inside `node_modules/.prisma/client` get updated:
```
Comment (janpio) [inline content/200-concepts/050-overview/100-what-is-prisma/index.mdx]:
```suggestion
After you change your data model, you'll need to manually re-generate Prisma Client to ensure the code inside `node_modules/.prisma/client` gets updated:
```

File: content/200-concepts/100-components/02-prisma-client/000-working-with-prismaclient/010-generating-prisma-client.mdx [hunk 5]
```diff
@@ -34,28 +34,28 @@ To generate and instantiate Prisma Client:
...
+   const { PrismaClient } = require('@prisma/client')
+   const prisma = new PrismaClient()
+   // use `prisma` in your application to read and write data in your DB
+   ```
 
-    </tab>
+   </tab>
 
-    </TabbedContent>
+   </TabbedContent>
```
Comment (janpio) [inline content/200-concepts/100-components/02-prisma-client/000-working-with-prismaclient/010-generating-prisma-client.mdx:58]: Are we sure changing this indentation does not break the formatting in the rendered version?

File: content/200-concepts/100-components/02-prisma-client/000-working-with-prismaclient/010-generating-prisma-client.mdx [hunk 6]
```diff
@@ -123,7 +123,7 @@ import { PrismaClient } from '../prisma/src/generated/client'
 
 #### Importing Prisma Client
 
-By generating Prisma Client into `node_modules/.prisma/client` and exporting it from `node_modules/@prisma/client`, you can import it and instantiate the client in your code as follows:
+By generating Prisma Client into `node_modules/.prisma/client` and exporting it from `node_modules/.prisma/client`, you can import it and instantiate the client in your code as follows:
```
Comment (janpio) [inline content/200-concepts/100-components/02-prisma-client/000-working-with-prismaclient/010-generating-prisma-client.mdx]:
```suggestion
By generating Prisma Client into `node_modules/.prisma/client` and exporting it from `@prisma/client`, you can import it and instantiate the client in your code as follows:
```

I think this better capture what this wants to say: We export into that folder, but use the `@prisma/client` package to import which does the redirect under the hood.

File: content/200-concepts/100-components/02-prisma-client/index.mdx [hunk 7]
```diff
@@ -67,7 +67,7 @@ You can install Prisma Client in your project with this command:
 npm install @prisma/client
 ```
 
-Note that this command also run the `prisma generate` command which generates the Prisma Client into the `node_modules/@prisma/client` directory.
+Note that this command also run the `prisma generate` command which generates the Prisma Client into the `node_modules/.prisma/client` directory.
```
Comment (janpio) [inline content/200-concepts/100-components/02-prisma-client/index.mdx]: "into the `node_modules/.prisma/client` directory." should link to an in depth explanation here, or it is just confusing here (And should be left off).

File: content/400-reference/200-api-reference/200-command-reference.mdx [hunk 8]
```diff
@@ -348,7 +348,7 @@ Watching... /home/prismauser/prisma/prisma-play/prisma/schema.prisma
 
 #### Generated Assets
 
-The `prisma-client-js` generator creates a customized client for working with your database within the `./node_modules/@prisma/client` directory.
+The `prisma-client-js` generator creates a customized client for working with your database within the `./node_modules/.prisma/client` directory.
```
Comment (janpio) [inline content/400-reference/200-api-reference/200-command-reference.mdx]: Should this also refer to the customization possible via `output`?

**Comment by janpio** [comment]
No worries, this is actually a great opportunity to clean up all these mentions of the location and clarify if they should be there, if they should link to an in depth explanation and so on. I left you a bunch of comments and suggestions.

File: content/200-concepts/100-components/02-prisma-client/000-working-with-prismaclient/010-generating-prisma-client.mdx (hunk 5 above)
Comment (molebox) [reply content/200-concepts/100-components/02-prisma-client/000-working-with-prismaclient/010-generating-prisma-client.mdx:58]: This is the prettier config we have, it runs an on commit hook. I can disable it with `--no-verify` but that kind of defeats the point. I guess the config must have changed from the last time someone made changes to this file and committed them.

Comment (mhwelander) [reply content/200-concepts/100-components/02-prisma-client/000-working-with-prismaclient/010-generating-prisma-client.mdx:58]: 👋 This is fine - if you make sure you double `<TabbedContent>` and friends, Prettier only reduces four spaces to three and leaves it alone after that.

Comment (janpio) [reply content/200-concepts/100-components/02-prisma-client/000-working-with-prismaclient/010-generating-prisma-client.mdx:58]: And this renders correctly under the list item?

Comment (mhwelander) [reply content/200-concepts/100-components/02-prisma-client/000-working-with-prismaclient/010-generating-prisma-client.mdx:58]: Yep, it does.

**Comment by mhwelander** [comment]
> Looking at the conflicts, they are pretty trivial so you should be able to fix them via GH UI directly @molebox. I am missing the knowledge which of these links and link titles are not the correct ones.

I fixed these conflicts. ✅

File: content/200-concepts/050-overview/100-what-is-prisma/index.mdx (hunk 3 above)
Comment (mhwelander) [reply content/200-concepts/050-overview/100-what-is-prisma/index.mdx]: Added link to https://www.prisma.io/docs/concepts/components/prisma-client/working-with-prismaclient/generating-prisma-client#the-prismaclient-npm-package

**Pull request merged**